c10/util/strong_type.h

```diff
@@ -33,20 +33,12 @@
 #endif
 
 #ifndef STRONG_HAS_STD_FORMAT
-#if __has_include(<format>)
-#define STRONG_HAS_STD_FORMAT 1
-#else
 #define STRONG_HAS_STD_FORMAT 0
 #endif
-#endif
 
 #ifndef STRONG_HAS_FMT_FORMAT
-#if __has_include(<fmt/format.h>)
-#define STRONG_HAS_FMT_FORMAT 1
-#else
 #define STRONG_HAS_FMT_FORMAT 0
```

Comment:
Disabling format ensures that we don't define min/max macros on Windows. No one appears to be using this functionality anyway. cc @robieta